app/src/main/res/layout/administrator_controls_download_permissions_view.xml

```diff
@@ -119,7 +119,7 @@
         style="@style/Subtitle2"
         android:background="@color/white"
         android:text="@string/administrator_controls_auto_update_message"
-        android:textColor="@color/divider"
+        android:textColor="#555555"
```

Comment:
Use this here too: `accessible_light_grey `

Reply:
done